### PR TITLE
count new line char

### PR DESCRIPTION
**Pull request by alcharkov**
This will count new lines, but will skip the first one. I need this feature, because on backend I get different length.

### CODE REVIEW

File: jquery.charactercounter.js [hunk 1]
```diff
@@ -87,6 +88,11 @@
         function checkCount(element)
         {
             var characterCount = $(element).val().length;
+
+	    if (options.countNewLineChars) {
+		characterCount+=$(element).val().split(/\r*\n/).length0;
```
Comment (dtisgodsson) [inline jquery.charactercounter.js]: What is `length0`? A typo?

**Comment by dtisgodsson** [comment]
@alcharkov so it doesn't count the first new line char?

File: jquery.charactercounter.js (hunk 1 above)
Comment (alcharkov) [reply jquery.charactercounter.js]: yes, sorry.

**Comment by alcharkov** [comment]
No, because in other case it would count even totally emty textbox

**Comment by dtisgodsson** [comment]
@alcharkov ok, I'll review this when I'm home tonight and merge it in if all good.

Thank you for your contribution! :)

**Comment by alcharkov** [comment]
Thanks you for library. I'm using it at my work 👍

**Comment by darrentaytay** [comment]
@alcharkov with your change, if I create a counter with a limit of 150, it starts off at 149 even when empty. 

I'll need to revert this unless we can get it sorted.

**Comment by darrentaytay** [comment]
image

**Comment by alcharkov** [comment]
Yes please, revert it. I've issues with mine client side also

**Comment by alcharkov** [comment]
Textarea has different new line chars depending which browser is used: http://blog.sarathonline.com/2009/09/javascript-maxlength-for-textarea-with.html

**Pull request merged**